android/app/src/main/AndroidManifest.xml

```diff
@@ -11,6 +11,7 @@
         android:icon="@mipmap/ic_launcher">
         <activity
             android:name=".MainActivity"
+            android:screenOrientation="portrait"
```

Comment:
Don't forget about other platforms. If you want to set a portrait orientation everywhere, you can set it natively at all of the platforms, not android only, or set orientation inside Flutter app.

Reply:
Replaced with
SystemChrome.setPreferredOrientations([
      DeviceOrientation.portraitUp,
    ]);